specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/examples/DataExportCreateOrUpdate.json

```diff
@@ -25,7 +25,6 @@
             "resourceId": "/subscriptions/192b9f85-a39a-4276-b96d-d5cd351703f9/resourceGroups/OIAutoRest1234/providers/Microsoft.EventHub/namespaces/test",
             "type": "EventHub"
           },
-          "allTables": false,
```

Comment:
why do you need to delete this field? This change would impact Azure CLI as well. Please share more context about this change

Reply:
We had a discussion with OBO service which is using this API. They needs to know the specific table names which should be exported, otherwise they can be break. So we decided to delete "AllTables" field so customer must supply a list a table names that they want to exported. 
I am aware of the change CLI issue. I will change it too after you will approve it.

Reply:
Even if customers use `AllTables`, since server would create tables for them, why doesn't OBO know the table names? It would be a breaking change for customers.

Reply:
OBO cannot do that.
We are still in preview mode. Our PM in contact with all the customers that use it. They already got notice with this change so they are aware.